mender/cli/utils.py

```diff
@@ -20,6 +20,13 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import requests
+from urllib.request import urlretrieve
+
+def download_image(uri, store=False):
```

Comment:
I think this should be part of deployments cli and use requests. You can try this:

```
from shutil import copyfileobj

def download_image(url, store=False, **kwargs):
    rsp = do_simple_get(url, stream=True, **kwargs)
    logging.debug('status %s', rsp.status_code)
    if rsp.status_code == 200:
        if store:
            with open('update-image.ext') as outf:
                copyfileobj(rsp.raw, outf)
    else:
        logging.error('failed to download image from %s: %s', url, rsp.text)
````